app/models/compute_resources/foreman/model/openstack.rb

```diff
@@ -129,30 +129,27 @@ def user_data_supported?
 
     private
 
+    def fog_credentials
+      { :provider => :openstack,
+        :openstack_api_key  => password,
+        :openstack_username => user,
+        :openstack_auth_url => url,
+        :openstack_tenant   => tenant,
+        :openstack_identity_endpoint => url }
```

Comment:
is there an openstack setup which we need to allow the user to provide that url separately to the auth_url ? e.g do we ever need to expose it to the user?